tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -89,15 +94,15 @@ describe(__filename, () => {
     delete partialAddon.homepage;
     delete partialAddon.support_email;
     delete partialAddon.support_url;
-    const root = render({ addon: createInternalAddon(partialAddon) });
```

Comment:
I guess this was serving the purpose of initializing `support_url` to an empty string (or whatever). Initializing to empty props seems like a good idea so maybe put this back?

Reply:
The problem here is that `partialAddon` is already an internal add-on object, so calling `createInternalAddon` and passing it in doesn't really make sense. `createInternalAddon` is only supposed to accept an `ExternalAddonType`.

Reply:
It makes sense to call `createInternalAddon()` a second time because several properties were deleted. The `createInternalAddon()` function will normalize those properties (I think).

Reply:
As discussed on IRC, adding this back in does nothing.